events/_single.html

```diff
@@ -24,6 +24,23 @@
 {% block content_main %}
     {% import "_event.html" as article with context %}
     {{ article.render(vars.mock_event, vars.path) }}
+
+    <div class="event-footer">
+        <p class="event-footer_banner">
+            <span class="event-footer_banner_message">This event requires an RSVP. Reserve your spot here: </span>
```

Comment:
Not sure which is BEM correct here, but looking at https://github.com/cfpb/capital-framework/issues/40#issuecomment-36789156